packages/docker-engine/docker-engine.spec

```
@@ -3,9 +3,9 @@
 %global goorg github.com/docker
 %global goimport %{goorg}/docker
 
-%global gover 20.10.18
+%global gover 20.10.20
 %global rpmver %{gover}
-%global gitrev e42327a6d3c55ceda3bd5475be7aae6036d02db3
+%global gitrev 3056208812eb5e792fa99736c9167d1e10f4ab49
```

This is the gitrev for `20.10.21`.
```suggestion
%global gitrev 03df974ae9e6c219862907efdd76ec2e77ec930b
```